test_bot/test_bot.py

```diff
@@ -185,7 +183,7 @@ def run_bot(raw_config: dict[str, Any], logging_level: int, opponent_path: str =
     config.insert_default_values(raw_config)
     CONFIG = config.Configuration(raw_config)
     lichess_bot.logger.info(lichess_bot.intro())
-    li = lichess_bot.lichess.Lichess(CONFIG.token, CONFIG.url, lichess_bot.__version__)
+    li = lichess_bot.test_bot.lichess.Lichess(CONFIG.token, CONFIG.url, lichess_bot.__version__, logging_level, 1)

```

Comment:
Is there a reason for not using `import testbot.lichess` and `li = test_bot.lichess.Lichess()`?